ci/build.ps1

```diff
@@ -26,6 +27,12 @@ function exitIfFailed() {
   }
 }
 
+function exitAtEndIfFailed() {
+  if ($exitAtEnd -eq 0 -And $LastExitCode -ne 0) {
+    $exitAtEnd = $LastExitCode
```

Comment:
I think it's good practice to specify `$script:` scope, not sure why we don't already do that.

    $script:exitAtEnd = $LastExitCode

Reply:
Hmm, I do not feel like messing with this now (using multiple attempts likely).. ;)